Alt Text and Link Edits - Pt. 4 (iOS) (#3342)
braze-inc/braze-docs: _docs/_developer_guide/platform_integration_guides/ios/advanced_use_cases/google_tag_manager.md

Braze's iOS SDK can be initialized and controlled by tags configured within [Google Tag Manager][5].

> But first - before using Google Tag Manager - be sure to first follow our [initial SDK setup][1].

Reviewer: ```suggestion
Before using Google Tag Manager, be sure to first follow our [initial SDK setup][1].
```

## Configuring your Google Tag Manager {#configuring-ios-google-tag-manager}

> In this example, we’ll pretend we are a music streaming app that wants to log different events as users listen to songs. Using Google Tag Manager for iOS, we can control which of our 3rd party vendors receive this event and create tags specific to Braze.

Reviewer: ```suggestion
In this example, we’ll pretend we are a music streaming app that wants to log different events as users listen to songs. Using Google Tag Manager for iOS, we can control which of our third-party vendors receive this event and create tags specific to Braze.
```

### Custom events

Custom events are logged with `actionType` set to `logEvent`. The Braze custom tag provider in our example is expecting the custom event name to be set using `eventName`.

To get started, create a trigger that looks for an "Event Name" that equals `played song`

![A custom trigger in Google Tag Manager set to trigger for some events when "event name" equals "played song".][3]

Next, create a new Tag ("Function Call") and enter the class path of your [custom tag provider](#adding-ios-google-tag-provider) described later in this article. 

This tag will be triggered when you log the `played song` event we just created. 

In our example tag's custom parameters (key-value pairs), we've set `eventName` to `played song` - which will be the custom event name logged to Braze.

{% alert important %}

> When sending a custom event, set `actionType` to `logEvent`, and set a value for `eventName` as shown in the screenshot below. 

Reviewer: ```suggestion
When sending a custom event, set `actionType` to `logEvent`, and set a value for `eventName` as shown in the following example. 
```
